### ISSUE TITLE
cosmos theme - learning objectives

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#3175]
# Version 1 (LO + Chapter Outline)
![Screen Shot 2020-12-11 at 5 41 31 PM](https://user-images.githubusercontent.com/11299125/101964819-431ed180-3bd8-11eb-9cf2-8c8f74c933c0.png)

## Notes
- These learning objectives are in the introduction and are merged with the chapter outline.
- They should be able to break across to the second page
- Should be flush to the left like regular content, but cut it at 5.5in wide to match the intro splash title

## Title
font: Archivo
weight: bold, 700
treatment: all caps
font size: font-scale(0): 9pt, 12px, 1rem
color: BR1: `#8B693A`

## Module number
weight: bold, 700
font: IBM Plex Sans
font size: font-scale(0): 9pt, 12px, 1rem
color: black `#000000`

## Module title
font: IBM Plex Sans
font size: font-scale(0): 9pt, 12px, 1rem
color: black `#000000`

## List
should follow basic nested list specs
font: IBM Plex Sans
font size: font-scale(0): 9pt, 12px, 1rem
color: black `#000000`

**Comment by ajluk** [comment]
@lwtchu  Here is how Learning Objectives look styled in American Government: 
![image](https://user-images.githubusercontent.com/55203598/104924727-20819480-599e-11eb-9275-83f01aa26fc2.png)

**Comment by lwtchu** [comment]
@ajluk this is good thank you

**Comment by ajluk** [comment]
@lwtchu we noticed that there is some inconsistency in the Learning Objectives markup in books in cosmos theme. Some of them has the text "By the end of this section.." grabbed in paragraph  - principles of Economics:
![image](https://user-images.githubusercontent.com/55203598/107029169-e4786d00-67ae-11eb-99f5-9dc761649947.png)
and in some of them it is just a text node instead - History, Psychology: 
![image](https://user-images.githubusercontent.com/55203598/107029446-4fc23f00-67af-11eb-8d0a-92cdf201a435.png)
Natalia pointed out during review of my PR that in carnival theme there was a content change in similar case. The text nodes were wrapped in paragraphs, but there this content change was necessary because this text/paragraph was needed to by styled differently than the list below. In cosmos the text and the lists have the same properties. 
What I did in styles I have styled the whole container to cover the cases with text nodes. But if you think we need that change in the content for consistency please let me know so I will rework the shape.

**Comment by lwtchu** [comment]
@ajluk i think i can request a content change to put the text in a `<para>` tag. Is it currently just history and psychology?

**Comment by ajluk** [comment]
Yes, it is just History and Psychology, in American Government there are already paragraphs, but different markup (section.learning-objectives).

**Comment by lwtchu** [comment]
@ajluk i requested the content change here: https://github.com/openstax/content-managers/issues/2101 for katalyst01 to get the change sometime this week. i'll let you know when it's done. do you think it's blocking your work in an adverse way?

**Comment by ajluk** [comment]
@lwtchu Thank you! No it's not a blocker for me, I have added a temporary support to this kind of markup, but please let me know when the content will be fixed, I will clean the code the and open a card for Marta to test it afterwards. I hope that is also ok by you :relaxed:

**Comment by lwtchu** [comment]
@ajluk looking at the CM card, it looks like the katalyst01 one for US history was fixed.

**Comment by lwtchu** [comment]
@ajluk jk i think theyre both done. thank you @Jacqueline-Yarbrough

**Comment by Jacqueline-Yarbrough** [comment]
@ajluk @wilkus27 yes both are done!

**Comment by kswanson33** [comment]
![image](https://user-images.githubusercontent.com/26280712/111665707-fe12c880-87e0-11eb-8039-e773d0dd2264.png)

Version 4: added `h-spacing(1)` between number and description
